scss/_variables.scss

```diff
@@ -696,7 +696,6 @@ $btn-padding-x: from-pixels(24) !default;
 $btn-font-family: $input-btn-font-family !default;
 $btn-line-height: 20 / 14 !default;
 $btn-letter-spacing: $input-btn-letter-spacing !default;
-$btn-text-transform: $input-btn-text-transform !default;
```

Comment:
Instead of deleting these lines, let's update $input-btn-text-transform value to be `default`